server/src/app/ghost/dto/query.param.dto.ts

```diff
@@ -0,0 +1,5 @@
+export class QueryParamDto {
+  gene: string;
```

Comment:
It seems good to add validation annotation like a `@IsEmpty()`!
[class-validator](https://github.com/typestack/class-validator) support this annotation.